### ISSUE TITLE
Events: Available data and status

### ISSUE THREAD

**Issue by jensneuber** [trustlines-network/relay#7]
As described in https://github.com/trustlines-network/clientlib/blob/develop/docs/Events.md it would be nice to have data on an event in the communication between jslib and app: 

```
{
  transactionId,  // transaction id on the blockchain, optional since transaction might be pending
  timestamp,      // unix timestamp
  subject,        // (optional)
  eventtype,      // one on the eventtypes defined below
  address,        // (proxy?)address of the counter party
  direction,      // send | receive 
  status,         // e.g. pending | done ... 
  amount
}
```

- Can the relay server determine sth like a status on a transaction?
- Is there a reliable timestamp?

**Comment by dakingha69** [comment]
- Can the relay server determine sth like a status on a transaction?
I talked to @berndbohmeier and we decided to go for the status `sent`, `pending`, `confirmed`.
`sent` - as soon as the transaction is pending (blockNumber and blockHash is null)
`pending` - transaction has at least 1 confirmation
`confirmed` - transaction hast at 5 (?) confirmations

- Is there a reliable timestamp?
We can get a unix timestamp from the blockNumber.
